waspc/cli/Main.hs

```diff
@@ -124,6 +125,7 @@ printDbUsage =
         "",
         title "EXAMPLES",
         "  wasp db migrate-dev",
+        "  wasp db migrate-dev migration-name",
```

Comment:
ah yes you are right, I mixed it up! But also agree it makes sense to use more "real" name instead of migration-name, although I think the context of this being an example (which I missed) is already enough. But might be fun to provide some interesting migration name, e.g. sth with spaces if that is supported? Just to immediately educate the user on what is possible. E.g. "Added User entity".